### PR TITLE
Google-clone

### PR DESCRIPTION
**Pull request by Stevenkwizera06**
###What does this PR do?
shows the cloning of the homepage of google

###Description of Task to be completed?
the Google word needs to stay always at the center of the page as the page expands or shrinks

###How should this be manually tested?

Clone the repo
- checkout to branch ft-dev
- run npm run dev

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
       <a href="#" class="hidden md:block text-white text-xsm hover:underline">Gmail</a>
       <a href="#" class="hidden md:block text-white text-xsm hover:underline">Images</a>
       <div class="hover:bg-primary_gray p-2 rounded-full">
-        
+
         <svg class="w-6 fill-secondary_dark" viewBox="0 0 24 24">
```
Comment (MugaboDenys) [inline index.html]: Here you should try to use svg from a file not codes

**Review by MugaboDenys — CHANGES_REQUESTED** [review]

**Review by MugaboDenys — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  <main class="flex flex-col mt-16 items-center  flex-1">
+    <div class="w-full flex flex-col gap-6 items-center ">
+      <img class="w-40 md:w-72" src="./images/light google.png" alt="Google logo">
+      <div
+        class="w-full flex items-center gap-2 border border-gray-300 max-w-xl rounded-full px-4 py-1 hover:shadow-md hover:bg-zinc-700 hover:border">
+
+        <img class="w-6 h-6 fill-dark_gray" src="./svgs/search.svg" alt="">
+        <div class="hidden md:flex space-x-2 ">
+          <input type="text" class="w-96 border-none outline-none text-white text-base bg-transparent ml-12">
+          <div class="flex ">
```
Comment (Ntwali-Josue) [inline index.html]: Try to imitate the spacing and the sizes of the original google input icons as close as you can.

your design: 
image

Google design: 
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        </div>
+      </div>
+      <div class="hidden md:block space-x-2">
+        <button type="button"
+          class="text-sm px-5 py-2 cursor-pointer hover:bg-zinc-800 text-white rounded  bg-zinc-700 border border-zinc-700 shadow-sm">Google
+          Search</button>
+        <button type="button"
+          class="text-sm px-5 py-2 cursor-pointer hover:bg-zinc-800 text-white rounded  bg-zinc-700 border border-zinc-700 shadow-sm">I'm
+          Feeling Lucky</button>
+      </div>
```
Comment (Ntwali-Josue) [inline index.html]: Changing the background of these buttons once they are hovered on, is not really needed, only change the border styles, once a button is hovered.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+  <main class="flex flex-col mt-16 items-center  flex-1">
+    <div class="w-full flex flex-col gap-6 items-center ">
+      <img class="w-40 md:w-72" src="./images/light google.png" alt="Google logo">
+      <div
+        class="w-full flex items-center gap-2 border border-gray-300 max-w-xl rounded-full px-4 py-1 hover:shadow-md hover:bg-zinc-700 hover:border">
+
+        <img class="w-6 h-6 fill-dark_gray" src="./svgs/search.svg" alt="">
+        <div class="hidden md:flex space-x-2 ">
+          <input type="text" class="w-96 border-none outline-none text-white text-base bg-transparent ml-12">
```
Comment (Ntwali-Josue) [inline index.html]: Try to reduce the size of the input field and also add a camera icon on the mobile devices, check the differences between your design and the original one

Yours: 
image
Google: 
image

File: images/micro.html [hunk 5]
```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
```
Comment (Ntwali-Josue) [inline images/micro.html]: You can remove this file, since it's not being used.

**Review by Ntwali-Josue — CHANGES_REQUESTED** [review]
Hey, @Stevenkwizera06 Check down below the reviews on your work.

**Review by Ntwali-Josue — APPROVED** [review]

File: tailwind.config.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+        primary_dark_blue: '#1a73e8',
+        secondary_dark_blue: '#8ab4f8',
+        secondary_bg_dark:'#0a0a0a',
+        light_gray: '#f8f9fa',
+      },
+      fontFamily: {
+        Roboto: ['Roboto', 'sans-serif'],
+      },
+      fontSize: {
+        xsm: '0.813rem',
```
Comment (Morrism1) [inline tailwind.config.js]: Can you check how Tailwind name their content and follow it accordingly.

**Review by Morrism1 — CHANGES_REQUESTED** [review]
Looks Good. One change in the config.

**Review by Morrism1 — APPROVED** [review]
LGTM!

**Pull request merged**